src/main/java/org/jitsi/jigasi/transcription/TranscriptionResultPublisher.java

```diff
@@ -34,4 +34,12 @@
      * @param result the result
      */
     void publish(ChatRoom chatRoom, TranscriptionResult result);
+
+    /**
+     * Publish the given TranslationResult to the given ChatRoom
+     *
+     * @param chatRoom the chat room
+     * @param result the result
+     */
+    void publish(ChatRoom chatRoom, TranslationResult result);
```

Comment:
A `TranslationResultPublisher` interface can help in separating the options to send translated text and the normal text from sip-communicators.properties file.

Reply:
All results are being spammed in the same Chatroom. Probably a different UI element is to be used for different languages?

Reply:
yes, or we filter on the preferred language and only show results in that language